configs/12.0/packages/gcc/sources

```diff
@@ -25,7 +25,7 @@ ATSRC_PACKAGE_SUBPACKAGE_NAME_1="GNU Libgomp"
 ATSRC_PACKAGE_VER=8.2.1
 ATSRC_PACKAGE_REV=266159
 ATSRC_PACKAGE_LICENSE="GPL 3.0"
-ATSRC_PACKAGE_DOCLINK="https://gcc.gnu.org/onlinedocs/gcc-${ATSRC_PACKAGE_VER}/gcc/"
+ATSRC_PACKAGE_DOCLINK="https://gcc.gnu.org/onlinedocs/gcc-8.2.0/gcc/"
```

Comment:
What about `${ATSRC_PACKAGE_VER/%.[0-9]/.0}` instead?
We should also add a comment here explaining why this is necessary.

Reply:
@tuliom Good catch. I've updated my patch and added a comment.